web/src/OverviewItemView.tsx

```diff
@@ -244,6 +254,24 @@ export function triggerUpdate(name: string, action: string) {
   })
 }
 
+export function toggleTriggerMode(name: string, mode: TriggerMode) {
+  incr("ui.web.toggleTriggerMode", { to_mode: mode.toString() })
+
+  let url = `//${window.location.host}/api/override/trigger_mode`
+
+  fetch(url, {
+    method: "post",
+    body: JSON.stringify({
+      manifest_names: [name],
```

Comment:
i thought all our json serialization stuff defaulted to camelCase, as in this file:
https://github.com/tilt-dev/tilt/blob/master/web/src/view.d.ts

i worry this is going to really bite us later if half our code uses camelCase and half our code uses slug case, depending on who wrote it that day.

Reply:
Thanks for calling this out. I'm updating this to consistently use camelCase.

Reply:
@nicks [all the hud api endpoints use snake case](https://github.com/tilt-dev/tilt/blob/57dbb09cac112a251bbb2547fc085d9358333579/internal/hud/server/server.go#L48), so at least we're consistent within api endpoints and within typescript stuff, so I feel fine leaving this for now. (there might be a case to be made for converting the api endpoints to camelCase but that seems like a conversation separate from this epic.)